README.md

```diff
@@ -17,3 +18,20 @@ In order to run the MIMIC-IV ETL, simply run the following command:
 In order to run the MIMIC-IV ETL, simply run the following command:
 
 `meds_etl_omop omop omop-esds` where omop is a folder containing csv files (optionally gzipped) for an OMOP dataset. Each table should either be a csv file with the table name (such as person.csv) or a folder with the table name containing csv files.
+
+## Unit tests
+
+Tests can be run from the project root with the following command:
+
+```
+pytest -v
+```
+
+Tests requiring data will be skipped unless the `tests/data/` folder is populated first. 
+
+To download the testing data, run the following command/s from project root:
```

Comment:
We probably want to make a script for this, as we will want to also include synthetic OMOP data, but this is good for now.

Reply:
I agree, this could be improved. I wasn't entirely sure of the best way of implementing tests on local machines. I don't really like the idea of local tests requiring access to external resources, but this is an option (e.g. when you run the test, you pull down data from HuggingFace, PhysioNet, etc).

Reply:
I think we could package the mimic demo data in this repository (maybe just 1 patient to save space)? Or generate synthetic MIMIC format data in the same way that I have synthetic OMOP data?

Reply:
Either of these would be good with me. II was cautious about adding too much volume to the repo, but just picking a few patients makes sense. I love the idea of a synthetic data generator.